cg/models/slurm/sbatch.py

```diff
@@ -13,6 +15,6 @@ class Sbatch(BaseModel):
     email: str
     hours: int
     minutes: str = "00"
-    priority: Literal["high", "normal", "low"] = "low"
+    priority: Literal[SlurmQos.LOW, SlurmQos.NORMAL, SlurmQos.HIGH] = SlurmQos.LOW
```

Comment:
```suggestion
    priority: SlurmQos = SlurmQos.LOW
```
ref: https://pydantic-docs.helpmanual.io/usage/types/#enums-and-choices